transport/src/androidMain/kotlin/com/genesys/cloud/messenger/transport/util/TokenGenerator.kt

```diff
@@ -0,0 +1,7 @@
+package com.genesys.cloud.messenger.transport.util
+
+import java.util.*
+
+actual object TokenGenerator {
+    actual fun generate() = UUID.randomUUID().toString()
+}
```

Comment:
Lint will also complain on missing new line in the end of the file.
you can run the `./gradlew lintKotlin`  command to verify.

Reply:
Thanks for the `lintKotlin` tip. Fixed.